src/ferny/__init__.py

```diff
@@ -30,3 +30,5 @@
     'temporary_askpass',
     'write_askpass_to_tmpdir',
 ]
+
+__version__ = '0'
```

Comment:
No :(

This is probably the most-requested feature of flit and it's been turned down time and time again.

This turned into a case of choosing your poison, but there were lots of reasons to avoid setuptools/setuptools_scm and this was the only reason I might have avoided flit.  Meanwhile, none of the other tools are any better....

Reply:
Unfortunately not.  flit absolutely requires a `__version__` in your `__init__.py`.  It's one of those "opinionated" things.

Reply:
I slowly understand all these people complaining about Python's build systems..